### DISCUSSION TITLE
Where can I buy this book? Is there an amazon link?

### DISCUSSION THREAD

**Discussion by vhenwan** [halfrost/LeetCode-Go#245] [Q&A] [answered]
As the title says

**Comment by halfrost** [comment, marked as answer]
I'm so sorry, this book has not been published yet. I will inform you when it is published. Thank you for your support. If you don't mind, you can read the pdf version first. The download link for the pdf version can be found on the [release page](https://github.com/halfrost/LeetCode-Go/releases).

**Reply by vhenwan** [reply]
Hope to publish soon.
